### PR TITLE
Re-use script from osi repo

### PR DESCRIPTION
**Pull request by ClemensLinnhoff**
#### Reference to a related issue in the repository
#55
#### Add a description

- [ ] Replace osi_trace.py with OSITrace.py from open-simulation-interface submodule
- [ ] General refactoring

#### Mention a member
Add @mentions of the person or team responsible for reviewing proposed changes.

#### Check the checklist

- [ ] My code and comments follow the [contributors guidelines](https://opensimulationinterface.github.io/osi-documentation/osi/howtocontribute.html) of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have made corresponding changes to the [documentation](https://github.com/OpenSimulationInterface/osi-documentation) for osi-validation.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests / travis ci pass locally with my changes.

### CODE REVIEW

File: osivalidator/osi_general_validator.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
     import osi_rules
     import osi_validator_logger
     import osi_rules_checker
-    import osi_trace
+    from format.OSITrace import OSITrace
 except Exception as e:
     print(
         "Make sure you have installed the requirements with 'pip install -r requirements.txt'!"
```
Comment (jdsika) [inline osivalidator/osi_general_validator.py:20]: Filename is outdated. We have splitted into requirements needed for develping and the ones for running the software.

Comment (ClemensLinnhoff) [reply osivalidator/osi_general_validator.py:20]: But this is part of the open-simulation-interface python package now. What do you mean by "Filename is outdated"?

Comment (jdsika) [reply osivalidator/osi_general_validator.py:20]: Maybe I am confused but that looks like the try except for the import of dependedncies for the osi-validation and not osi (sub module) with the failing message to install the requirements.txt (which does not exist in osi-validation)?

Comment (ClemensLinnhoff) [reply osivalidator/osi_general_validator.py:20]: Ah sorry, your comment was about the error message. I thought it was about line 17. My mistake. Yes, I'll fix the message.

**Pull request closed without merging**